tests/acceptance/SettingsPageBasicsCest.php

```diff
@@ -61,17 +61,17 @@ public function allowSubscribeInComments(\AcceptanceTester $i) {
     $i->click($postTitle);
     $i->waitForText($postTitle);
     $i->scrollTo('#commentform');
+    $i->scrollTo('.comment-form-comment');
     $i->waitForElementVisible(['css' => '.comment-form-mailpoet']);
-    $i->see($optinMessage);
+    $i->waitForText($optinMessage);
     $i->click('#comment');
     $i->fillField('#comment', $comment);
     $i->waitForText($optinMessage);
     $i->click('#mailpoet_subscribe_on_comment');
     $i->click('Post Comment');
     $i->waitForText($comment, 10, '.comment-content');
     //check if user is really subscribed to a list
-    $i->amOnMailpoetPage('Lists');
-    $i->clickItemRowActionByItemName('Newsletter mailing list', 'View Subscribers');
+    $i->amOnMailpoetPage('Subscribers');
```

Comment:
This makes the test weaker. If the subscriber ends up in a wrong list the test still passes. How does it help with the flakiness?

Reply:
Yes, you're totally right! Why I changed that to be more weak, only God knows. Now I strengthed it. Thanks!